rs/nns/governance/canister/governance.did

```diff
@@ -11,7 +11,7 @@ type Action = variant {
   SetDefaultFollowees : SetDefaultFollowees;
   RewardNodeProviders : RewardNodeProviders;
   ManageNetworkEconomics : NetworkEconomics;
-  ApproveGenesisKyc : Principals;
+  ApproveGenesisKyc : Principals_1;
```

Comment:
Can we fix this before we merge this?  These kinds of changes are not backwards compatible for clients that use this to generate code (yet)

Reply:
Ok, I added a commit that fixes it

Reply:
Fixed!